lib/releases.js

```diff
@@ -157,6 +166,7 @@ const generateChangeLog = (mergedPullRequests, config) => {
       return acc
     }, [])
     .join('')
+    .trim()
```

Comment:
Removes all the empty newlines at the end, I noticed during testing if I added even more categories, it would result in more empty newlines.

Reply:
properly better to check
```javascript
if (index + 1 !== orderedCategories.length && categorizedPullRequests[category].length) acc.push('\n\n')
```

Reply:
Never mind, trim() is the correct thing since it is done as part adding pull requests.